cmd/server/main.go

```diff
@@ -9,13 +9,18 @@ import (
 	"math/big"
```

Comment:
Not really in the scope of this PR, but linter reports the following warning:
`cmd/server/main.go:7:2: `io/ioutil` is in the denylist: io/ioutil is deprecated. Use package io or os instead. (depguard)`